src/pytest/test.py

```diff
@@ -411,28 +405,28 @@ def testCustomStopwords(env):
     r = env
     # Index with default stopwords
     env.assertOk(r.execute_command(
-        'ft.create', 'idx', 'schema', 'foo', 'text'))
+        'ft.create', 'idx', 'ON', 'HASH', 'FILTER', 'startswith(@__key, "")', 'schema', 'foo', 'text'))
 
     # Index with custom stopwords
-    env.assertOk(r.execute_command('ft.create', 'idx2', 'stopwords', 2, 'hello', 'world',
+    env.assertOk(r.execute_command('ft.create', 'idx2', 'ON', 'HASH', 'FILTER', 'startswith(@__key, "")', 'stopwords', 2, 'hello', 'world',
                                     'schema', 'foo', 'text'))
     if not env.isCluster:
         res = env.cmd('ft.info', 'idx2')
         env.assertEqual(res[39], ['hello', 'world'])
 
     # Index with NO stopwords
-    env.assertOk(r.execute_command('ft.create', 'idx3', 'stopwords', 0,
+    env.assertOk(r.execute_command('ft.create', 'idx3', 'ON', 'HASH', 'FILTER', 'startswith(@__key, "")', 'stopwords', 0,
                                     'schema', 'foo', 'text'))
     
     if not env.isCluster:
         res = env.cmd('ft.info', 'idx3')
         env.assertEqual(res[39], [])
 
-    for idx in ('idx', 'idx2', 'idx3'):
-        env.assertOk(r.execute_command(
-            'ft.add', idx, 'doc1', 1.0, 'fields', 'foo', 'hello world'))
-        env.assertOk(r.execute_command(
-            'ft.add', idx, 'doc2', 1.0, 'fields', 'foo', 'to be or not to be'))
+    #for idx in ('idx', 'idx2', 'idx3'):
```

Comment:
?

Reply:
We don't need to loop since each index has a rule that indexes the hash